client/src/pages/spools/create.tsx

```diff
@@ -49,10 +50,17 @@ export const SpoolCreate: React.FC<IResourceComponentsProps & CreateOrCloneProps
     return {
       label: label,
       value: item.id,
+	    weight: item.weight,
+	    spool_weight: item.spool_weight
     };
   });
   filamentOptions?.sort((a, b) => a.label.localeCompare(b.label, undefined, { sensitivity: "base" }));
 
+  const [used_weight, set_used_weight] = useSavedState("create-used_weight", 0);
```

Comment:
This should be a regular react useState. `useSavedState` is a special version of it that I created in which the browser remembers the previous state, which is not something we want here.

Reply:
Yeah all you need to do is remove the first string argument and it works the same way.